src/compiler/checker.ts

```diff
@@ -9376,6 +9379,17 @@ namespace ts {
                     exportedType.callSignatures,
                     exportedType.constructSignatures,
                     exportedType.indexInfos);
+                if (initialSize === members.size) {
+                    if (type.aliasSymbol) {
+                        result.aliasSymbol = type.aliasSymbol;
+                        result.aliasTypeArguments = type.aliasTypeArguments;
+                    }
+                    if (getObjectFlags(type) & ObjectFlags.Reference) {

```

Comment:
Generally, arbitrary type reference types don't usually get alias symbols, but we transform it into an alias symbol here to improve printback, since our copy strips the reference-ness of the type.